frontend/app/equity/dividend_computations/[id]/page.tsx

```
@@ -0,0 +1,332 @@
+  );
+}
+
+export default function DividendComputationDetailPage() {
+  return (
+    <EquityLayout>
+      <Suspense fallback={<div>Loading...</div>}>
+        <DividendComputationDetailContent />
+      </Suspense>
+    </EquityLayout>
+  );
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)